README.md

```diff
@@ -10,57 +10,48 @@ see [2024-03-13](meetings/2024/2024-03-13.md) and [the backlog](meetings/backlog
 
 ## Meetings
 
-Meetings are bi-weekly, alternating between an APAC-friendly time and an EMEA-friendly time. The meetings appear on the [TC39 private calendar](https://github.com/tc39/Reflector#tc39-private-calendar).
+We meet weekly.
+The meetings appear on the [TC39 private calendar](https://github.com/tc39/Reflector#tc39-private-calendar).
 
-Meeting link: <https://meet.google.com/rwh-opnw-cnk>
+Meeting link: <https://agoric.zoom.us/j/7965396534?pwd=UlJ6aXZLdnVwdlZJRko4SjBtQ1dZdz09>
```

Comment:
@ctcpip and i decided to go with https://us02web.zoom.us/j/81143085896?pwd=TUE3WGgrdEZmNFZJc0g4QzBHUWczdz09 for now (on my personal paid zoom account), which is what's on the tc39 calendar invite, but we can discuss further if needed.

Reply:
Updated to your room. I’ll have to keep a foot in the above room for 5 minutes at the front of each meeting for a while to make sure we don’t leave stragglers.